Controller/Payment/Test.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
Please remove this file `Controller/Payment/Test.php` which is not require.

Reply:
done